src/routes/(console)/project-[region]-[project]/updateVariables.svelte

```diff
@@ -480,3 +483,10 @@
         {variableList}
         bind:show={showVariablesUpload} />
 {/if}
+
+{#if showDeleteModal}
+    <DeleteVariableModal
+        bind:show={showDeleteModal}
+        {selectedVar}
+        onDelete={handleVariableDeleted} />
+{/if}
```

Comment:
We can use the `Confirm` component here directly for any type of pre-confirmation.